app/org/sagebionetworks/bridge/services/ParticipantService.java

```diff
@@ -91,50 +92,71 @@ final void setExternalIdService(ExternalIdService externalIdService) {
     final void setCacheProvider(CacheProvider cacheProvider) {
         this.cacheProvider = cacheProvider;
     }
+
+    /**
+     * This StudyParticipant will always include a healthCode and will never include consent histories. 
+     * It is used to initialize a user session and should not be returned directly through the API. 
+     * It avoids calling Stormpath again to initialize the StudyParticipant, since an account is 
+     * returned on authentication. The method getParticipant() enforces permissions on who can see 
+     * consent and healthCode information. 
+     */
+    public StudyParticipant getParticipantForSession(Study study, Account account) {
+        checkNotNull(study);
+        checkNotNull(account);
+        
+        StudyParticipant.Builder builder = getParticipantBuilder(study, NO_CALLER_ROLES, account);
+        builder.withHealthCode(account.getHealthCode());
+        return builder.build();
+    }
```

Comment:
Added a test that verifies healthCode is included in the call, along with some other fields you'd expect from full initialization of the StudyParticipant.